twitch/api/users.py

```diff
@@ -12,6 +12,10 @@ class Users(TwitchAPI):
     def get(self):
         response = self._request_get('user')
         return User.construct_from(response)
+      
+    def get_by_name(self, user_id):
+        response = self._request_get('users?login={}'.format(user_id))
+        return User.construct_from(response)
```

Comment:
The documentation says that it returns multiple users (or one user inside a list), so you need to extract each user and convert it to User object
```
 return [User.construct_from(x) for x in response['users']]
```

Reply:
Yes you are right. That looks good.